cloud/deploys/bin/azure/lib.sh

```diff
@@ -31,6 +31,23 @@ function azure::create_vault() {
     --enable-rbac-authorization
 }
 
+#######################################
+# Assign the role 'Key Vault Secrets officer' to the current user
+# Arguments:
+#   1: The resource group name for the key vault
+#   2: The region (e.g. EastUS) to create the key vault in
+#   3: The name of the key vault 
+#######################################
+function azure::assign_secrets_officer_role_to_vault() {
```

Comment:
did you mean `assign_secrets_officer_role_to_user()`?

Reply:
eee yes that is better